About me.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<!-- saved from url=(0032)http://127.0.0.1:5500/about.html -->
+<html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+        
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <title>About me</title>
+        
+      <link rel="stylesheet" href="./About me_files/style.css"></head>
+
+      <body>
+        <header>
+          <nav>
+            <img src="./About me_files/profile logo.png" width="40" alt="profile logo">
+            <a class="new-about active" home="" href="http://127.0.0.1:5500/index.html">Home</a>  
+            <a class="new-about" href="http://127.0.0.1:5500/about.html">About</a>
```

Comment:
Why are you using a `http://127.0.0.1:5500` for linking internal files? It will fail in Github pages or any other hosting provider. Since its a file you have in your local machine or folder just add a reference to the file path. Eg: ` <a class="new-about" href="about.html">About</a>`